app/javascript/experiments/unintended-donations.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  experimentId: 'wXDP1ynuTQKHs6vVvMEj1A',
```

Comment:
Once the testing is done on staging, I will update the experiment id with the production Id